### ISSUE TITLE
[albums] Access by Author

### ISSUE THREAD

**Issue by plegall** [Piwigo/Piwigo#26]
**Reported by VDigital on 26 Dec 2006 10:27**

Author have to be managed.
auth_id is mandatory to avoid all difference from time to time.
Access as a category or a tag access has to be implemented.
A link between member and Author has to be defined.
Maybe later a role has to be created as vimages explained.

Topic related is http://forum.phpwebgallery.net/viewtopic.php?id=5693

[Piwigo Bugtracker #607](http://piwigo.org/bugs/view.php?id=607)

**Comment by plegall** [comment]
**Comment posted by VDigital on 31 Jan 2007 07:57**

Acked (without h), just to promote it.
I don&#039;t know if we have time enough to do it in 1.7

**Comment by plegall** [comment]
**Comment posted by plg on 2 Feb 2007 16:16**

&gt; I don&#039;t know if we have time enough to do it in 1.7

No, we don&#039;t, but you&#039;re right to promote this feature request. We have to make it right after branch 1.7 fork (on trunk, of course).

**Comment by plegall** [comment]
This feature request is a bit confused and mixes several ideas:

1) converting piwigo_images.author from a textfield into a list of values => we implemented it for search in Piwigo 2.6, see 0511ed2236b1fab9bc9c23b6c3bbee82ee4efdb5 => now we should go further and implement it "the clean way", with a piwigo_authors table, numeric id and so on.

2) browsing photos by author => once step 1 is done, this could be implemented with a plugin

3) having a relation between users and authors => this was implemented years ago with piwigo_images.added_by. We know which user has added which photo.
